core/domain/stats_services.py

```diff
@@ -68,7 +82,22 @@ def get_visualizations_info(exploration_id, state_name):
 
 # TODO(bhenning): Test
 def get_versions_for_exploration_stats(exploration_id):
-    """Returns list of versions for this exploration."""
+    """Returns a list of string, each string representing a version of the given
+    exploration_id for which statistics data exits. These versions are retrieved
+    from ExplorationAnnotationsModel created when StatisticsAggregator job is
+    run.
+
+    An example of the return list may look like [u'3', u'all']
+    where '3' and 'all' are versions of the given exploration ID from
+    ExplorationAnnotationsModel.
+
+    Args:
+        exploration_id: str. The exploration ID.
+
+    Returns:
+        list(str). A list of strings, each string representing version of the
```

Comment:
list(str). The versions of the given exploration for which statistics data exists. These may either be 'all' (which indicates that the statistics have been aggregated over all versions), or specific (stringified) version numbers.